content/docs/command-reference/remote/add.md

```diff
@@ -125,34 +125,40 @@ So, make sure you have the following permissions enabled:
 
 <details>
 
-### Click for S3 API compatible storage
+### Click for S3-compatible storage
 
-To communicate with a remote object storage that supports an S3 compatible API
+To communicate with a remote object storage that supports an S3-compatible API
 (e.g. [Minio](https://min.io/),
 [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/),
 [IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.),
-configure the remote's `endpointurl` explicitly:
-
-For example:
+configure the remote's `endpointurl`:
 
 ```dvc
 $ dvc remote add -d myremote s3://mybucket/path
 $ dvc remote modify myremote endpointurl \
                     https://object-storage.example.com
 ```
 
-> See `dvc remote modify` for a full list of S3 API parameters.
+It's also important to setup appropriate authentication with any of the related
```

Comment:
How about: 

⚠️ It's important to setup appropriate authentication with `dvc remote modify`. Otherwise, DVC uses default AWS credentials which may cause an error.

Reply:
Good rewrite! I'll resubmit it as an actionable GH suggestion and tag you. Resolving here